sstables/sstables_manager.hh

```diff
@@ -71,6 +71,7 @@ public:
 
     storage_manager(const db::config&, config cfg);
     shared_ptr<s3::client> get_endpoint_client(sstring endpoint);
+    bool has_endpoint_client(sstring endpoint) const;
```

Comment:
Nitpick: such a const function shouldn't take sstring by value (a copy), it's wasted effort - it should either take a `const sstring&`, or if the hash table implementation allows that, a std::string_view. But I'm mentioning this as a nitpick and not an important thing to fix because I see that the pre-existing get_endpoint_client() function has the same issue.

Reply:
Agree, `std::string_view` is much better. I'll fix them both in a followup.